wdtk-rdf/src/main/java/org/wikidata/wdtk/rdf/RdfConverter.java

```diff
@@ -474,4 +490,17 @@ boolean hasTask(int task) {
 		return ((this.tasks & task) == task);
 	}
 
+	String rankToRdfRepresentation(StatementRank rank) {
+		switch (rank) {
+		case NORMAL:
+			return "Normal";
+		case PREFERRED:
+			return "Preferred";
+		case DEPRECATED:
+			return "Deprecated";
+		default:
+			return null;
```

Comment:
Maybe throw an IllegalArgumentException. I don't think the outer code can recover from this anyway.